org-gantt.el

```diff
@@ -1478,6 +1478,25 @@ PARAMS determine several options of the gantt chart."
             (insert body))
           )))))
 
+(defun org-insert-dblock:org-gantt-chart ()
+  "Insert org-gantt dynamic block."
+  (interactive)
+  (org-create-dblock
+   (list :name "org-gantt"
+         :file "data/images/project-gantt-chart.png"
```

Comment:
maybe use a parameter here?

Reply:
I don't understand, what parameter you mean?

Reply:
the filename `"data/images/project-gantt-chart.png"` shouldn't be hard coded

Reply:
Hmm, I don't know what variable should be used here. Do you have any suggestion?

Reply:
sorry for the late reply. could you just add a parameter to the function? maybe one which is asked for if not provided? what about just calling it `filename` then?

Reply:
Ok, I force pushed the updated with optional `filename` parameter. Check it out, thanks for review. :)